### PR TITLE
Adding known instance for the 'InnerSource Portal' pattern

### PR DESCRIPTION
**Pull request by spier**
Adding known instance for the 'InnerSource Portal' pattern from Elbit Systems.
Looking for confirmation that this entry is fine to be shared this way.

Also added a direkt file like to the Gig Marketplace, and moving that sentence to a new section called **References**.

### CODE REVIEW

File: patterns/2-structured/innersource-portal.md [hunk 1]
```diff
@@ -47,12 +47,19 @@ The InnerSource Portal has enabled InnerSource project owners to advertise their
...
-* The InnerSource Portal pattern has been proven to work extremely well with the associated InnerSource Gig Marketplace pattern in this context
+* SAP promotes InnerSource projects in the InnerSource portal - projects can self-register using GitHub topics. The [Repository Activity Score](repository-activity-score.md) defines the default order of the InnerSource projects in the portal.
+* Elbit Systems has used this pattern and added gamification on top.
+	* [Gamification As Means of Cultural Change and InnerSource Engagement Booster](https://www.oreilly.com/library/view/oscon-2018-/9781492026075/video321579.html) | Shelly Nizri | OSCON 2018 - Portland, Oregon
+	* Of Islands, Monsters & InnerSource [(slides)](https://docs.google.com/presentation/d/1P1OCEK9B6eSrVRUclVWY6meSI-qHOBjM_UAPNvCZamU/edit#slide=id.p15), [(video)](https://drive.google.com/file/d/1pM89uHMn0vhE3ayFJDGYcCO8R0tAXXZD/view?usp=drivesdk) | InnerSource Spring Summit 2019 (Galway, Ireland)
+
```
Comment (lenucksi) [inline patterns/2-structured/innersource-portal.md:60]:
```suggestion
	* The code realizing this platform has been open sourced and is availble here: https://gitlab.com/gilda2

```

File: patterns/2-structured/innersource-portal.md [hunk 2]
```diff
@@ -47,12 +47,19 @@ The InnerSource Portal has enabled InnerSource project owners to advertise their
...
+## References
+
+* The InnerSource Portal pattern has been proven to work extremely well with the associated InnerSource [Gig Marketplace](./gig-marketplace.md) pattern in this context
```
Comment (lenucksi) [inline patterns/2-structured/innersource-portal.md:63]: We should also collect evidence where this combination has worked well. Pull requests very welcome :smile:

**Review by lenucksi — APPROVED** [review]
Thanks for creating this, @shelly-goldblit will likely want to sign-off on this.

File: patterns/2-structured/innersource-portal.md (hunk 2 above)
Comment (spier) [reply patterns/2-structured/innersource-portal.md:63]: Those stories would then go to "Known Instances" as well, right?

**Comment by shelly-goldblit** [comment]
The links to the slides and the video recording go to my personal google drive :grimacing:
Perhaps it would be best to create a copy?

File: patterns/2-structured/innersource-portal.md (hunk 2 above)
Comment (lenucksi) [reply patterns/2-structured/innersource-portal.md:63]: Could do. We should however mark that they serve as evidence for the combination of Portal+Gig Marketplace not the Portal pattern.

**Comment by ShellyDCMS** [comment]
The links to the slides and the video recording go to my personal google drive :grimacing:
Perhaps it would be best to create a copy?

**Comment by lenucksi** [comment]
> The links to the slides and the video recording go to my personal google drive grimacing
> Perhaps it would be best to create a copy?

Sure, happy to add them to the ISCF GDrive. Would that work for you? Are there any rights restrictions we would need to observe?

**Comment by ShellyDCMS** [comment]
> > The links to the slides and the video recording go to my personal google drive grimacing
> > Perhaps it would be best to create a copy?
> 
> Sure, happy to add them to the ISCF GDrive. Would that work for you? Are there any rights restrictions we would need to observe?

Regarding the slides: The rights belong to Elbit Systems Ltd. (it's stated at the bottom of each slide). I have an OK in writing to republish them here. 
Regarding the video recording, I am not sure who has the rights on that. It was recorded during the 2019 April ISC summit.

**Comment by lenucksi** [comment]
> 
> 
> > > The links to the slides and the video recording go to my personal google drive grimacing
> > > Perhaps it would be best to create a copy?
> > 
> > 
> > Sure, happy to add them to the ISCF GDrive. Would that work for you? Are there any rights restrictions we would need to observe?
> 
> Regarding the slides: The rights belong to Elbit Systems Ltd. (it's stated at the bottom of each slide). I have an OK in writing to republish them here.
> Regarding the video recording, I am not sure who has the rights on that. It was recorded during the 2019 April ISC summit.

That sounds great. I've asked around regarding the video recording. Let's see what comes out of this.

File: patterns/2-structured/innersource-portal.md (hunk 2 above)
Comment (spier) [reply patterns/2-structured/innersource-portal.md:63]: That will be certainly nice when we find such combinations.
I am not aware of these combinations, so for the time being I would suggest we close this thread?

**Comment by spier** [comment]
I resolved the conflicts between this branch and the changes to the **Known Instances** of this pattern from `master`.

This PR should be good to be merged now.

**Review by lenucksi — APPROVED** [review]
Nice, thanks for the additions.

**Pull request merged**